library/lib.sh

```diff
@@ -116,13 +125,14 @@ $(cat "$playbook_file")" > "$playbook_file".tmp
 rolesInstallDependencies() {
     local coll_req_file="$1/meta/collection-requirements.yml"
     local coll_test_req_file="$1/tests/collection-requirements.yml"
+    collection_path=$(mktemp --directory -t collections-XXX)
     for req_file in $coll_req_file $coll_test_req_file; do
         if [ ! -f "$req_file" ]; then
             rlLogInfo "Skipping installing dependencies from $req_file, this file doesn't exist"
         else
             rlRun "ansible-galaxy collection install -p $2 -vv -r $req_file"
-            rlRun "export ANSIBLE_COLLECTIONS_PATHS=$2"
-            rlLogInfo "Dependencies were successfully installed"
+            rlRun "export ANSIBLE_COLLECTIONS_PATH=$2"
```

Comment:
This is tricky.  Older versions of ansible use `ANSIBLE_COLLECTIONS_PATHS` and newer versions use `ANSIBLE_COLLECTIONS_PATH`.  This is how we determine which one to use in runqemu.py https://github.com/linux-system-roles/tox-lsr/blob/main/src/tox_lsr/test_scripts/runqemu.py#L84 - basically, run `ansible-config list` and see which one is available.

Reply:
Thanks for pointing, fixed